test_conformance/commonfns/test_mix.cpp

```diff
@@ -142,18 +161,33 @@ int test_mix_fn(cl_device_id device, cl_context context, cl_command_queue queue,
         test_error(err, "clCreateBuffer failed");
     }
 
-    for (i = 0; i < num_elements; i++)
+    std::string pragma_str;
+    if (std::is_same<T, double>::value)
```

Comment:
We're already setting this sting a couple of lines later, should this be checking for half and setting the half pragma?

Reply:
Done

Reply:
Corrected the same way as `smoothstep` - due to the fact that accuracy of `mix` for `cl_khr_fp16` is implementation defined this test should only report maximum error without testing maximum error threshold